src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api/getPeole';
+import PeopleTable from '../../components/PeopleTable/PeopleTable';
+import { IPeople, IPeopleWithParents } from '../../entities/IPeople';
+
+function PeoplePage() {
+  const [people, setPeople] = useState<IPeopleWithParents[]>();
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    const humans = async () => {
+      const peoples = await getPeople();
+
+      return peoples;
+    };
+
+    humans()
+      .then((result) => {
+        const peoplesWithParents: IPeopleWithParents[] = result.map((child: IPeople) => {
+          const mother = result.find((person: IPeople) => person.name === child.motherName);
+          const father = result.find((person: IPeople) => person.name === child.fatherName);
+
+          return (
+            {
+              ...child,
+              mother,
+              father,
+            }
+          );
+        });
```

Comment:
Also, you can create getPeoplesWithParents method and make easier your useEffect